src/System-Settings-Core/SettingDeclaration.class.st

```diff
@@ -181,7 +182,7 @@ SettingDeclaration >> ghostHelp: aString [
 	ghostHelp := aString
 ]
 
-{ #category : 'testing' }
+{ #category : 'accessing' }
```

Comment:
I think testing was a better protocol here

Reply:
This is weird because I see it already in 'testing' category:

<img width="787" alt="Screenshot 2024-04-04 at 20 36 06" src="https://github.com/pharo-project/pharo/assets/4825959/2e978aea-b135-4299-9161-a6d7fe8a6a7a">